tests/AuthTest.php

```diff
@@ -53,8 +62,8 @@ public function testActingAs()
     {
         $mockedUser = new MockAuthUser();
 
-        $this->actingAs($mockedUser);
+        $this->testCaseMock->actingAs($mockedUser);
 
         $this->assertNotSame($mockedUser, Auth::user());
```

Comment:
it will always be true as the `Auth::user` is the user from the currect app, and `mockedUser` is just a variable which setting inside the mock class